WebAutopark/Vehicle/VehicleNameComparer.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WebAutopark
+{
+    public class VehicleNameComparer : IComparer<Vehicle>
```

Comment:
Why do you need this class?

Reply:
I left this class from the previous project. Already removed